src/test/kotlin/com/timcastelijns/room15bot/usecases/MaukerUseCaseTest.kt

```diff
@@ -0,0 +1,21 @@
+package com.timcastelijns.room15bot.usecases
+
+import com.timcastelijns.room15bot.bot.usecases.MaukerUseCase
+import org.junit.Test
+import kotlin.test.assertTrue
+
+class MaukerUseCaseTest {
+
+    private val reply = "Mauker, plz."
+
+    @Test
+    fun testMaukerReply() {
+        assertTrue { reply == MaukerUseCase().execute(null) }
```

Comment:
better use `assertEquals(expected, actual)` I used to use assertTrue but have recently started switching.